File: AMR15A/cpp/main.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
    cout << "text" << endl;
    int N;
    int noOfLucky = 0;
    int noOfOdd = 0;
    cin >> N;
    for(int i = 0; i < N; i++){
        int weapons;
        cin >> weapons;
        cout << "weapons " << weapons << endl;
        if(weapons % 2 == 0){
            cout << "even" << endl;
            noOfLucky++;
        } else {
            cout << "odd" << endl;
            noOfOdd++;
        }
        cout << "even " << noOfLucky << " odd " << noOfOdd << endl;
    }

    if (noOfLucky > noOfOdd){
        cout << "READY FOR BATTLE" << endl;
    } else {
        cout << "NOT READY" << endl;
    }

	return 0;
}

File: AMR15A/main.go
package main

import (
	"fmt"
	"log"
)

func main() {
	//fmt.Println("main")
	var N, noOfLucky, noOfOdd int
	fmt.Scan(&N)
	for i := 0; i < N; i++ {
		log.Println("N -> ", N)
		var weapons int
		fmt.Scan(&weapons)
		if weapons%2 == 0 {
			//log.Println("even")
			noOfLucky = noOfLucky + 1
		} else {
			//log.Println("odd")
			noOfOdd = noOfOdd + 1
		}

	}
	log.Println(noOfLucky, noOfOdd)
	if noOfLucky > noOfOdd {
		fmt.Println("READY FOR BATTLE")
	} else {
		fmt.Println("NOT READY")
	}
}
